pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "acapy_wallet_groups_plugin"
-version = "0.1.6"
+version = "0.1.7"
```

Comment:
Hmmm, I understand the big version bump now. Would be interested in why it was already 0.1.6

Reply:
i guess because it picks the version from poetry toml here and not the other one? Or it picks the highest git tag? which was 0.1.6

Reply:
since the ci/cd is using poetry publish i guess it's using the version from pyproject.toml?!